samigo/samigo-app/src/webapp/jsf/section-activity/sectionActivityHeadings.jsp

```diff
@@ -8,9 +8,9 @@
        <h:outputText value="#{generalMessages.assessment}" />
        <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.author.AuthorActionListener" />
     </h:commandLink>
-<f:verbatim></span></li>
+<f:verbatim rendered="#{authorization.adminTemplate and template.showAssessmentTypes}"></span></li>
```

Comment:
> That’s close for the preceding item and open for the next. 

It's closing a span and an li tag..... Why not place a whole logical section into a panelGroup that is controlled with a rendered attribute?